experimenter/experimenter/targeting/constants.py

```diff
@@ -53,6 +53,10 @@ def __post_init__(self):
 # 23:59 UTC on Dec 9, 2025 when an updated version of the privacy
 # notice was published.
 DEC_9_2025 = 1765324740000
```

Comment:
This constant is no longer being used, but I was unsure if it'd be ok to delete it. Happy to do so if that wouldn't cause any issues.